src/ui/public/state_management/state_monitor.js

```diff
@@ -50,6 +55,17 @@ function stateMonitor(state, defaultState) {
   };
 
   return {
+    setDefault(defaultState) {
+      // update the originalState and apply ignoredProps
+      if (defaultState) {
```

Comment:
I could make it throw instead, since you do need to pass something then. That work?